c/include/ml-api-service.h

```diff
@@ -224,9 +224,83 @@ int ml_service_query_create (ml_option_h option, ml_service_h *handle);
 int ml_service_query_request (ml_service_h handle, const ml_tensors_data_h input, ml_tensors_data_h *output);
 
 /**
- * @brief TBU
+ * @brief Registers a neural network model.
+ * @since_tizen 7.5
+ * @remarks Only one model will be activated with given @a key. If same key is already registered in machine learning service, this returns no error and old model will be deactivated when the flag @a activate is true.
+ * @param[in] key The unique key to indicate the model.
+ * @param[in] path The path to neural network model.
+ * @param[in] activate The flag to set the model to be activated.
+ * @param[in] description Nullable, description for neural network model.
+ * @param[out] version The version of registered model.
+ * @return 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Given parameter is invalid.
```

Comment:
I wonder if ML_ERROR_OUT_OF_MEMORY is necessary.

Reply:
IMO, unnecessary. This does not return new handle and passes args via dbus. Return value would be a json or  string and it does not need large memory.

Reply:
Thanks for the answer.